lib/worker_logs.js

```diff
@@ -1,6 +1,6 @@
-var fs = requrie('fs');
+var fs = require('fs');
```

Comment:
I'm not aware of what might be using this.  If this is not used anywhere I say remove it.